src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+or **Save**.
+
+{: #disable-submit}
+
+Consider disabling a submit button once the user has tapped or clicked it—especially when the user is 
+making a payment or placing an order. Many users click buttons repeatedly, even if they're working fine. 
+That can mess up checkout and add to server load. 
+
+On the other hand, don't disable a submit button waiting on complete and valid user input. For 
+example, don't just leave a **Save Address** button disabled because something is missing or invalid. 
+That doesn't help the user—they may continue to tap or click the button and assume that it's broken. 
+Instead, explain to the user what's gone wrong and what they need to do if they attempt to submit 
+the form with invalid data. This is particularly important on mobile, where data entry is more 
+difficult and missing or invalid form data may not be visible on the user's screen by the time they 
+attempt to submit a form.
+
+### Make the most of HTML attributes {: #html-attributes}
+
+#### Make it easy for users to enter data
+
+{: #type-attribute}
+
+Use the appropriate input [`type` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) 
+to provide the right keyboard on mobile and enable basic built-in validation by the browser. 
+
+For example, use `type="email"` for email addresses and `type="tel"` for phone numbers.
+
+<figure class="w-figure">
+  <img src="images/input-type.jpg" alt="Two screenshots of Android phones, showing a keyboard appropriate 
+  for entering an email addresss (using type=email) and for entering a telephone number (with type=tel).">
+  <figcaption class="w-figcaption">Keyboards appropriate for email and telephone.</figcaption>
+</figure>
+
+{: #inputmode-attribute}
+
+{% Aside 'warning' %}
+Using type="number" adds an up/down arrow to increment numbers, which makes no sense for data such as 
+telephone, payment card, or account numbers.
+
+Instead, you should set `type="text"` (or leave off the attribute, since `text` is the default) and 
+specify `inputmode="numeric"`, to get a numeric keyboard on mobile. 
+
+To ensure that users get the right keyboard, some sites still use `type="tel"` for payment card 
+numbers. However, `inputmode` is [very widely supported now](https://caniuse.com/input-inputmode), 
+so you shouldn't have to do that, but check your users' browsers.
+{% endAside %}
+
+{: #avoid-custom-elements}
+
+For dates, try to avoid using custom `select` elements. They break the autofill experience if not 
+properly implemented and don't work on older browsers. For numbers such as birth year, consider 
+using an `input` element rather than a `select`, since entering digits manually can be easier and less 
+error prone than selecting from a long drop-down list—especially on mobile. Use `inputmode="numeric"` 
+to ensure the right keyboard on mobile and add validation and format hints with text or a 
+placeholder to make sure the user enters data in the appropriate format.
```

Comment:
Why don't you recommend `input[type="date"]`?

Reply:
Hmm... Good point. I think mainly because that doesn't seem to be the norm. I'll take a look.